Makefile

```diff
@@ -19,7 +19,7 @@ _check_python:
 		fi;
 .PHONY: markdown-toc
 markdown-toc:
-	@if ! npm ls markdown-toc; then npm install; fi
+	@if ! npm ls markdown-toc; then npm ci; fi
```

Comment:
Took me a while to figure out what was happening, but this `npm install` was causing the `git diff` in the ToC check to detect changes. I guess this should take care of it?